### DISCUSSION TITLE
Show the option to create an item at the bottom, rather than top, of the suggestion dropdown?

### DISCUSSION THREAD

**Discussion by WhiteMac** [orchidjs/tom-select#575] [Ideas]
Currently suggestion lists look something like:

- Add {input}...
- Suggestion
- Suggestion

I'd like them to be:

- Suggestion
- Suggestion
- Add {input}...

Is there an option for this?

CSS flex ordering doesn't appear to be a sufficient workaround as it interferes with up/down arrow navigation of options.

Thank you.

**Comment by sanchawebo** [comment]
Furthermore i would like to see the option to set the position for the loading element.
